components/renku_data_services/migrations/env.py

```diff
@@ -68,30 +69,25 @@ def run_migrations_online(target_metadata, config: Config) -> None:
         with context.begin_transaction():
             context.run_migrations()
 
+    engine.dispose()
+
 
 def run_migrations(metadata: MetaData):
     """Run migrations for a specific base model class."""
     # this is the Alembic Config object, which provides
     # access to the values within the .ini file in use.
-    config = context.config
-
-    if not config.attributes.get("repo"):
-        config_class = config.get_section_option(config.config_ini_section, "config_class")
-
-        if config_class is None:
-            raise ValueError("Must set 'config_class' in alembic.ini section for app.")
-
-        config_module = importlib.import_module(config_class)
-
-        custom_config = config_module.Config.from_env()
-        config.attributes["repo"] = custom_config.repo
-
-    # Interpret the config file for Python logging.
-    # This line sets up loggers basically.
-    if config.config_file_name is not None:
-        fileConfig(config.config_file_name)
+    pg_host = os.environ.get("DB_HOST", "localhost")
+    pg_user = os.environ.get("DB_USER", "renku")
+    pg_port = os.environ.get("DB_PORT", "5432")
+    db_name = os.environ.get("DB_NAME", "renku")
+    pg_password = os.environ.get("DB_PASSWORD")
```

Comment:
i think it'd be nicer to still use the config class here, but with these properties as fields, so we don't refer to env vars in more than 1 place. we already have sync_db_connection_url anyways on the config object

Reply:
i guess we can also leave it. but it's also not nice having to remember to change two places if we need to change this. but it's just a small nitpick anyways